tests/unit/test_enum_element.py

```diff
@@ -0,0 +1,139 @@
+# Ported from square/wire:
+# wire-library/wire-schema/src/jvmTest/kotlin/com/squareup/wire/schema/internal/parser/EnumElementTest.kt
+
+from karapace.protobuf.enum_constant_element import EnumConstantElement
+from karapace.protobuf.enum_element import EnumElement
+from karapace.protobuf.kotlin_wrapper import trim_margin
+from karapace.protobuf.location import Location
+from karapace.protobuf.option_element import OptionElement
+
+location: Location = Location.get("file.proto")
+
+
+def test_empty_to_schema():
```

Comment:
Please always include the return type for functions, in this case it should be `-> None`. This is necessary because with mypy type checking is opt-in, to enable type checking at least one type in the function signature needs a type. Consistently typing the functions return type solves issues like this. It can be a real time saver while you're writing a test to have type checking enabled :)

Reply:
Could You provide me Your mypy config? I just installed it to my pycharm so would be nice to have the same config as Yours.

Reply:
I don't have a separate configuration. If you want to change the defaults you can add a [file `mypy.ini`](https://mypy.readthedocs.io/en/stable/config_file.html) in the repo :)

Reply:
will You add mypy to `make lint`?

Reply:
> will You add mypy to make lint?

Unfortunately not yet. It would make sense to add it as a `pre-commit` hook, but at the moment there are some errors that we have to fix before enabling it.